node/src/components/rpc_server/rpcs/docs.rs

```diff
@@ -236,6 +236,26 @@ impl OpenRpcSchema {
         required_params
     }
 
+    /// Convert the schema for the optional params type for T into the OpenRpc-compatible map of
+    /// name, value pairs.
+    ///
+    /// Since all params must be unanimously optional, mark all incorrectly tagged "required" fields
+    /// as false.
+    fn make_optional_params(schema: Schema) -> Vec<SchemaParam> {
+        let schema_object = schema.into_object().object.expect("should be object");
+        let optional_params = schema_object
```

Comment:
We don't really need the `optional_params` variable here.

Reply:
Removed variable